Simplenote.xcodeproj/project.pbxproj

```diff
@@ -1855,19 +1809,6 @@
 			shellPath = /bin/sh;
 			shellScript = "plist=\"${TARGET_BUILD_DIR}/${INFOPLIST_PATH}\"\nyear=$(date \"+%Y\")\ncopyright=\"Copyright © $year Automattic. All rights reserved.\"\n\n# Set the version numbers in the buildPlist\n#\n# Temporarily disabled! It's failing the build after updating to support Apple Silicon.\n#\n# Unfortunately, a number of changes were required to make the build pass before getting to this point,\n# so there is no clear responsible for this failure yet.\n# /usr/libexec/PlistBuddy -c \"Set :NSHumanReadableCopyright $copyright\" $plist\n";
 		};
-		B51D44911945FEFD00496889 /* ShellScript */ = {
-			isa = PBXShellScriptBuildPhase;
-			buildActionMask = 2147483647;
-			files = (
-			);
-			inputPaths = (
-			);
-			outputPaths = (
-			);
-			runOnlyForDeploymentPostprocessing = 0;
-			shellPath = /bin/sh;
-			shellScript = "# Codesign\n#\nIDENTITY=\"Developer ID Application: Automattic, Inc.\"\nSPARKLE_BIN=\"${SRCROOT}/External/Sparkle/bin\"\nCODESIGN_TOOL=\"${SPARKLE_BIN}/codesign_embedded_executable\"\nCODESIGN_HARDENEDRUNTIME_CMD=\"codesign --force -o runtime -s\"\n\n# XPC\n#\nXPC_SOURCE=\"${SRCROOT}/External/Sparkle/XPCServices\"\nXPC_TARGET=\"${BUILT_PRODUCTS_DIR}/${FRAMEWORKS_FOLDER_PATH}/../XPCServices\"\n\n# Services\n#\nXPC_NETWORK=\"org.sparkle-project.InstallerConnection.xpc\"\nXPC_LAUNCHER=\"org.sparkle-project.InstallerLauncher.xpc\"\nXPC_STATUS=\"org.sparkle-project.InstallerStatus.xpc\"\n\n# UPDATER\nUPDATER_TARGET=\"${BUILT_PRODUCTS_DIR}/${FRAMEWORKS_FOLDER_PATH}/Sparkle.framework/Versions/A/Resources\"\nSPARKLE_AUTOUPDATE=\"Autoupdate\"\nSPARKLE_UPDATER=\"Updater.app/Contents/MacOS/Updater\"\n\n# Bail out whenever we're not in Release\n#\nif [ \"${CONFIGURATION}\" != \"Release\" ]; then\n    echo \"NOT Signing in debug!\"\n    exit 0\nfi\n\n# Reset: Sparkle's Codesign fails when attempting to re-sign\n#\necho \"Nuking old XPC...\"\n\nrm -r \"${XPC_TARGET}\"\n\n# Copy Phase\n#\necho \"Copying...\"\n\nmkdir \"${XPC_TARGET}\"\ncp -R \"${XPC_SOURCE}/${XPC_NETWORK}\" \"${XPC_TARGET}/${XPC_NETWORK}\"\ncp -R \"${XPC_SOURCE}/${XPC_LAUNCHER}\" \"${XPC_TARGET}/${XPC_LAUNCHER}\"\ncp -R \"${XPC_SOURCE}/${XPC_STATUS}\" \"${XPC_TARGET}/${XPC_STATUS}\"\n\n# Codesign Phase\n#\necho \"Signing... \"\n\n${CODESIGN_TOOL} \"${IDENTITY}\" \"${XPC_TARGET}/${XPC_NETWORK}\"\n${CODESIGN_TOOL} \"${IDENTITY}\" \"${XPC_TARGET}/${XPC_LAUNCHER}\"\n${CODESIGN_TOOL} \"${IDENTITY}\" \"${XPC_TARGET}/${XPC_STATUS}\"\n${CODESIGN_HARDENEDRUNTIME_CMD} \"${IDENTITY}\" \"${UPDATER_TARGET}/${SPARKLE_AUTOUPDATE}\"\n${CODESIGN_HARDENEDRUNTIME_CMD} \"${IDENTITY}\" \"${UPDATER_TARGET}/${SPARKLE_UPDATER}\"\n";
-		};
```

Comment:
CocoaPods should be doing this for us now. See https://github.com/sparkle-project/Sparkle/issues/709.

Reply:
Yep, that's a huge win for sure!